components/fonts/xorg-cyrillic/Makefile

```diff
@@ -29,24 +29,19 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		font-misc-cyrillic
 COMPONENT_VERSION= 	1.0.3
-COMPONENT_REVISION=	1
+COMPONENT_REVISION=	2
 COMPONENT_ARCHIVE_HASH=	\
 	sha256:e40fe3e3323c62b738550795457ad555c70c008aa91b5912dfd46f8e745f5e60
 
-COMPONENT_NAME_2=	font-screen-cyrillic
-COMPONENT_VERSION_2= 	1.0.4
-COMPONENT_ARCHIVE_HASH_2=	\
-	sha256:824231e8dffe15299454e47259f29d98001c9cf8ad3d6b5171399e4d71705e79
-
 COMPONENT_NAME_3=	font-winitzki-cyrillic
 COMPONENT_VERSION_3= 	1.0.3
 COMPONENT_ARCHIVE_HASH_3=	\
 	sha256:abd13b63d02fcaec488686c23683e5cf640b43bd32f8ca22eeae6f84df0a36a0
 
 # Use font-screen-cyrillic version number as solaris-userland does
-IPS_COMPONENT_VERSION=$(COMPONENT_VERSION_2)
+IPS_COMPONENT_VERSION=$(COMPONENT_VERSION)
```

Comment:
This line does have two problems now:

- it no longer matches the comment above, and
- it will cause the package version decrease (1.0.4 -> 1.0.3).

I suggest to hardcode `IPS_COMPONENT_VERSION` to `1.0.4`.